pkgs/by-name/li/libhydrogen/package.nix

```
@@ -0,0 +1,66 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  testers,
+  pkg-config,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "libhydrogen";
+  version = "2024-04-06";
```

```suggestion
  version = "0-unstable-2025-04-07";
```
Since this commit is not a tagged release this should use the unstable naming scheme.